Makefile

```diff
@@ -140,23 +147,36 @@ OBJS += \
 	pow_fpga_accel.o
 endif
 
+ifeq ("$(BUILD_REMOTE)", "1")
+OBJS += \
+	remote_common.o \
+	remote_interface.o
+
+WORKER_OBJS := $(addprefix $(OUT)/remote-,$(filter-out remote_interface.o, $(OBJS)))
```

Comment:
The `$(OUT)/remote` can be changed to `$(OUT)/worker` to avoid naming confusion like **remote-remote_common.o**.